fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/IndexBuilderImpl.java

```diff
@@ -60,7 +72,13 @@ public void rebuild() {
 
         LOGGER.info("Initiating index rebuild.");
         fedoraToOCFLObjectIndex.reset();
+        final var transaction = transactionManager.create();
+        final var txId = transaction.getId();
+        final var rootFedoraId = FEDORA_ID_PREFIX;
```

Comment:
Probably don't need to re-define this string here.

Reply:
valid point.